src/components/team-entries/TeamEntries.tsx

```diff
@@ -0,0 +1,50 @@
+import { useContext } from "react";
+
+import { TeamEntriesContext } from "../../context/TeamEntriesProvider";
+import { Modal } from "../modal/Modal";
+import { TeamEntry } from "../team-entry/TeamEntry";
+import { TeamEntryForm } from "../team-entry-form";
+import * as Styled from "./TeamEntries.styled";
+
+interface TeamEntriesProps {
+  handleModal: () => void;
+  isModalActive: boolean;
+  currentClient: React.ComponentState;
+}
+
+export const TeamEntries = ({ isModalActive, handleModal, currentClient }: TeamEntriesProps) => {
+  const { teamEntries, setTeamEntries } = useContext(TeamEntriesContext);
+
+  return (
+    <>
```

Comment:
You can remove these fragments (`<></>`) since your container wraps the whole component.